platformcomponents/desktop/attachment-file-default.json

```diff
@@ -0,0 +1,21 @@
+{
+  "attachment-file-default": {
+    "sent": {
+      "comment": "Applied to sent & safe file attachments",
+      "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=5428%3A29",
+      "border": "@theme-outline-secondary-normal",
+      "iconForeground": "@theme-common-text-white",
+      "iconBackground": {
+        "default": "@theme-background-accent-normal"
+      },
+      "textFileName": "@theme-text-primary-normal",
+      "textFileSize": "@theme-text-secondary-normal",
+      "#normal": {
```

Comment:
Add all background states, please
```
"background": {
     "#normal": "@theme-background-secondary-normal",
     "#hovered": "@theme-background-secondary-hover",
     "#active": "@theme-background-secondary-active",
     "#focused": "@theme-background-secondary-normal"
}
```

Reply:
Pushed commit
`Added all background states for attachment-file-default`
Changing this conversation to Resolved.